positron/comms/data_explorer-backend-openrpc.json

```diff
@@ -5,6 +5,34 @@
 		"version": "1.0.0"
 	},
 	"methods": [
+		{
+			"name": "open_dataset",
+			"summary": "Request to open a dataset given a URI",
+			"description": "Request to open a dataset given a URI",
+			"params": [
+				{
+					"name": "uri",
+					"description": "The resource locator or file path",
+					"required": true,
+					"schema": {
+						"type": "string"
+					}
+				}
+			],
+			"result": {
+				"schema": {
+					"name": "open_dataset_result",
+					"type": "object",
+					"required": [],
+					"properties": {
+						"error_message": {
+							"description": "An error message if opening the dataset failed",
+							"type": "string"
+						}
+					}
+				}
+			}
+		},
```

Comment:
This new "open_dataset" request is useful, but will not be supported for now by the Python/R runtimes